packages/ifpack2/src/Ifpack2_Details_Chebyshev_def.hpp

```diff
@@ -1437,6 +1487,41 @@ powerMethodWithInitGuess (const op_type& A,
       return zero;
     }
     x.update (one / norm, y, zero);
+
+    std::ostringstream oss;
+    oss << std::scientific << std::setprecision(8) << "   lambda = " << lambdaMax;
```

Comment:
Theoretically. Practically, it is split among several `if()` statements, so I did not want to add a `debug_` check in all of them.